### ISSUE TITLE
Web ui symbols

### ISSUE THREAD

**Issue by RonnyRusten** [vitotai/BrewManiacEsp8266#64]
![image](https://user-images.githubusercontent.com/1955294/72850203-e1e0cd80-3ca8-11ea-9fc4-0a35dd1955f8.png)
* The symbol for Sparge heater is missing. It shows correct on LCD (the letter S inverted), but on web lcd it says 1.
* Degree Celsius symbol is "funky" on set value. The same error is on LCD.

**Comment by allthepies** [comment]
Additionally the spelling mistake on the panel title still exists "Cotnroller" should be "Controller"

**Comment by vitotai** [comment]
OK. I know this issue, but it would take some time to fix, excluding the "controller"one.

**Comment by RonnyRusten** [comment]
I have replaced all occurrences of "Cotnroller" that VS Code could find, but it still shows "Cotnroller".
If you could tell me where in code the error might be, I can look into it... also the "funky" symbols...

**Comment by RonnyRusten** [comment]
Sorry, I'm an idiot. 
The symbol behind 34.75 isn't the deg celcius, it's the SetPoint symbol...
The symbol under the Heat symbol when both Heat and Sparge Heat is activated is just an regular letter S, I guess, therefore it isn't in the definitions. If you turn off heat, it switches to a 1, which is defined as primary heat, which I believe is wrong, it should either be 2, or the inverse S, also, the inverse H should turn into a regular H.

**Comment by vitotai** [comment]
I thought I didn't consider some scenarios, but it turns out I did. This bug is solved by correcting the glyphs in Javascript. Try download bm.htm.gz from html.src/dist/english/ and try again. Don't forget to reload the page.

**Comment by RonnyRusten** [comment]
Yes, this worked for the symbols, but it still says "Cotnroller". I tried to change it in bm.htm.gz and upload again, but it didn't work. Is this defined somewhere else?

**Comment by vitotai** [comment]
Well. modifying the bm.htm.gz should fix it. You might need to reload again.

**Comment by RonnyRusten** [comment]
I finally got it to work, but I had to open the bm.htm.gz and change it directly in 7zip editor. How do I build a new bm.htm.gz? I have tried to change some of the other texts, but I can't figure out how this works. And, where does the other texts come from? I have changed texts in english.json, but nothing is changing on the web, and the date/time on bm.htm.gz isn't updated when I run the Build task.

**Comment by vitotai** [comment]
The building process of HTML/JS files is defined in grunt.coffee.
